config/redirects.js

```diff
@@ -1641,7 +1641,7 @@ module.exports = [
     to: '/flows/client-credentials-flow'
   },
   {
-    from: ['/flows/guides/auth-code/add-login-auth-code','/flows/guides/auth-code/includes/authorize-user-add-login','/flows/guides/auth-code/includes/sample-use-cases-add-login','/flows/guides/auth-code/includes/refresh-tokens','/flows/guides/auth-code/includes/request-tokens','/flows/guides/regular-web-app-login-flow/add-login-using-regular-web-app-login-flow','/oauth-web-protocol', '/protocols/oauth-web-protocol', '/protocols/oauth2/oauth-web-protocol','/application-auth/current/server-side-web','/client-auth/server-side-web','/application-auth/legacy/server-side-web'],
+    from: ['/flows/concepts/token-exchange','/flows/guides/auth-code/add-login-auth-code','/flows/guides/auth-code/includes/authorize-user-add-login','/flows/guides/auth-code/includes/sample-use-cases-add-login','/flows/guides/auth-code/includes/refresh-tokens','/flows/guides/auth-code/includes/request-tokens','/flows/guides/regular-web-app-login-flow/add-login-using-regular-web-app-login-flow','/oauth-web-protocol', '/protocols/oauth-web-protocol', '/protocols/oauth2/oauth-web-protocol','/application-auth/current/server-side-web','/client-auth/server-side-web','/application-auth/legacy/server-side-web'],
```

Comment:
The Token Exchange flow is a completely different flow than Auth Code. I wrote the docs for it last fall, but the project has been on hold since then. I believe they're picking it up some time next year again. But it also looks the like the part of the Dashboard UI that is directing people there is hidden unless your tenant has a specific feature flag set.

I'll defer to you since you know this Dashboard request, but in my opinion, this link should 404 since the docs don't exist. Redirecting to Auth Code seems like it might confuse some people?

Reply:
That makes total sense. I'll remove it.